07/fundamentals.html

```diff
@@ -185,33 +186,32 @@ <h2 id="flushing"><a href="#flushing">Message queueing, chunks and flushing</a><
     }
             </code>
         </pre>
-        And <code>m_Connection.QueueChunk()</code> as the name suggests only puts
-        these messages into a queue of so called chunks. So no packet header is created
+        As the name suggests, <code>m_Connection.QueueChunk()</code> only puts
+        these messages into a <em>queue</em> of so-called chunks; no packet header is created
```

Comment:
mr fancy pants with [&lt;em&gt;](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/em) tags, hyphens and semicolons lool

Shakespeare developer english